queries.sql

```diff
@@ -43,12 +43,17 @@ SET weight_kg = weight_kg * -1
 WHERE weight_kg <0;
 COMMIT;
 
-
-
 SELECT COUNT(*) FROM animals;
 SELECT COUNT(escape_attempts) FROM animals WHERE escape_attempts = 0;
 SELECT AVG(weight_kg) FROM animals;
 SELECT neutered, COUNT(escape_attempts) FROM animals GROUP BY neutered ORDER BY COUNT(escape_attempts) DESC LIMIT 1;
 SELECT species MAX(weight_kg), MIN(weight_kg) FROM animals GROUP BY species;
 SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN DATE '1990-01-01' AND '2000-01-01' GROUP BY species;
 
+SELECT * FROM animals a JOIN owners o ON o.id = a.owner_id WHERE o.full_name = 'Melody Pond';
+SELECT * FROM animals a JOIN species s ON s.id = a.species_id WHERE s.name = 'Pokemon';
```

Comment:
- [ ] Please we want to List all animals that are pokemon, it would be better to use LEFT JOIN here instead of inner join.

Reply:
This should be marked as OPTIONAL.

Reply:
Hi @AlucardSanin why do think this should be optional, here we want to choose just the animals that are owned by `What animals belong to Melody Pond?` you are doing an inner join I think it is not the same 😳
![image](https://user-images.githubusercontent.com/74744787/221241736-a3774d58-c85d-49e0-97e8-0a41f586b722.png)

Reply:
This query answer that question, if you wanted it to be displayed differently it's ok, with LEFT JOIN might be better approach, but with just JOIN works, so the use of LEFT JOIN instead of JOIN should be optional.
![image](https://user-images.githubusercontent.com/99287916/221244968-ff9521a8-9367-4951-8151-df8ed45b300e.png)